### DISCUSSION TITLE
Introduce yourself :wave:

### DISCUSSION THREAD

**Discussion by khaeru** [transport-data/community#2] [General]
Please leave a reply giving:
- Name,
- Affiliation,
- Link to any other official profile(s) (you can also add these to your GitHub profile)
- Interest in the TDC(I), and
- Current contributions to TDC(I) activities.

**Comment by gustavorinaldi-arg** [comment]
Name: Gustavo Rinaldi 

Argentina (Messi, Maradona, currently world champion =)) )

Affiliation: Expert in Environmental Law, Transport Policy, and Sustainability.

LinkedIn:  https://www.linkedin.com/in/rinaldigustavo/

**Interest in the TDC(I):**
My interest lies at the intersection of environmental law, transport policy, and climate sustainability. With over 15 years of experience in the public and academic sectors, I am focused on how high-quality open data can support emission reduction plans and improve access to climate finance. 
I aim to contribute expertise in strategic environmental management and the application of technologies, such as blockchain, to sustainability.
